src/NodaTime/LocalDate.cs

```diff
@@ -105,6 +105,7 @@ public LocalDate(int year, int month, int day)
         /// <param name="calendar">Calendar system in which to create the date.</param>
         /// <returns>The resulting date.</returns>
         /// <exception cref="ArgumentOutOfRangeException">The parameters do not form a valid date.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="calendar"/> is null.</exception>
```

Comment:
No need to do this - the fact that we have the NonNull annotation means these are already documented after generation. See https://nodatime.org/unstable/api/NodaTime.LocalDate.html#NodaTime_LocalDate__ctor_System_Int32_System_Int32_System_Int32_NodaTime_CalendarSystem_